website/src/pages/cwkw-placeholder.page.tsx

```diff
@@ -0,0 +1,16 @@
+import CWKWLayout from "src/cwkw/cwkw-layout"
+
+const CWKWPlaceholderPage = () => (
+  <CWKWLayout>
+    <main>
```

Comment:
Could you add here:

```tsx
<Helmet>
  <meta name=”robots” content=”noindex,nofollow”/>
</Helmet>
```

Helmet comes from `react-helmet`, you can look for other usages to see examples. This will prevent the page from being indexed since we don't want anyone to get linked to it.